src/wx/CMakeLists.txt

```diff
@@ -784,12 +784,15 @@ set(
 
 set(ALL_HDR_WX ${HDR_WX})
 
-list(APPEND ALL_SRC_WX winsparkle-wrapper.cpp)
```

Comment:
The reason I place these outside the if check is so that the gettext pot generation includes them.

Reply:
I duplicated the appends, just like before.